packages/react-client/src/__tests__/ReactFlight-test.js

```diff
@@ -185,6 +191,9 @@ describe('ReactFlight', () => {
           <ErrorBoundary expectedMessage="Symbol values (foo) cannot be passed to client components.">
             <Client transport={symbol} />
           </ErrorBoundary>
+          <ErrorBoundary expectedMessage="Refs cannot be passed from server components to client components.">
```

Comment:
Error message should be `Refs cannot be used in server components, nor passed to client components.`

Reply:
Ah yes. I originally didn't think of the case where you pass a ref to server component from another server component which also errors. Forgot to update the test.